components/pipeline-service/production/stone-prd-m01/deploy.yaml

```diff
@@ -1883,7 +2023,6 @@ spec:
     enable-git-resolver: true
     enable-hub-resolver: true
     enable-tekton-oci-bundles: true
-    enable-step-actions: true
```

Comment:
yeah pretty sure @amisstea recently enabled this .... something is inadvertently reverting this @enarha

Reply:
Same thing as above, different PR #3818 , changes the deploy.yaml files instead of the pipeline-service repo. I'll revert the changes to the deploy.yaml files and create a PR to update pipeline-service. @Omeramsc FYI.